src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/config/PersonalisationMappingConstants.java

```diff
@@ -79,7 +79,8 @@ public final class PersonalisationMappingConstants {
     public static final String LETTER_ADDRESS_LINE_2 = "letter_address_line_2";
     public static final String LETTER_ADDRESS_LINE_3 = "letter_address_line_3";
     public static final String LETTER_ADDRESS_LINE_4 = "letter_address_line_4";
-    public static final String LETTER_ADDRESS_POSTCODE = "letter_address_postcode";
+    public static final String LETTER_ADDRESS_POSTCODE = "letter_address_line_5";
```

Comment:
does this not still need to be letter_address_postcode for the templates?

Reply:
@nilay913 LETTER_ADDRESS_POSTCODE is only used by the tests is PdfLetterServiceTest.java. Changed to make consistent with other placeholders values used by templates.